content/handbook/engineering/infrastructure-platforms/developer-experience/_index.md

```
@@ -2,117 +2,117 @@
+DevEx チームは、自分たちの作業を加速し、また内側から GitLab の AI 機能を検証するために、AI ツールを積極的に活用しています。
 
-- [DevEx における AI の活用方法](ai/) — ツール、ガイドライン、ワークフロー、ドッグフーディングのプラクティス
-- [トップヒント](ai/top-tips/) — 日々のエンジニアリング業務のための実践的な AI ワークフロー
+- [DevEx で AI をどう使っているか](ai/) — ツール、ガイドライン、ワークフロー、ドッグフーディングのプラクティス
+- [トップヒント](ai/top-tips/) — 日常のエンジニアリングタスクに使える実践的な AI ワークフロー
 
-## 私たちとの協働
+## 私たちと一緒に働く
 
-各 [DevEx チーム](#team-structure) はハンドブックページにロードマップを維持し、トップレベルのチームエピックにリンクされたエピックに取り組みます。
+各 [DevEx チーム](#team-structure) は、自分たちのハンドブックページにロードマップを保持し、トップレベルチームエピックにリンクされたエピックに取り組みます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)